PMM-8446 move nginx role after system update

nikita-b opened the pull request.

BupycHuk reviewed (approved):
Could you please add comment how does this fixes the main problem?

nikita-b commented:
> Could you please add comment how does this fixes the main problem?
@BupycHuk 
We fixed two issues here:
1. We didn't have control on the nginx version in a container because we always had the latest version from epel repo.
2. We removed pmm.conf and pmm-ssl.conf from pmm-server package and if we reinstall the package files were gone and I moved nginx role AFTER reinstall pmm-server.


I'll add comment about it to ticket
